src/colorizer/state/slices/collection_slice.ts

```diff
@@ -0,0 +1,27 @@
+import { StateCreator } from "zustand";
+
+import { DatasetSlice } from "./dataset_slice";
+
+import Collection from "../../Collection";
+
+type CollectionSliceState = {
+  collection: Collection | null;
+};
+
+type CollectionSliceActions = {
+  setCollection: (collection: Collection) => void;
+  // TODO: Add action for loading and setting a dataset here
+};
+
+export type CollectionSlice = CollectionSliceState & CollectionSliceActions;
+
+export const createCollectionSlice: StateCreator<CollectionSlice & DatasetSlice, [], [], CollectionSlice> = (
+  set,
+  _get
```

Comment:
Do you need to include the explicit unused argument here or can you just leave it off?

Reply:
I can leave it off!

Reply:
Mm, decided to leave it because I expect this slice to need it in the near future